web/packages/shared/components/EmptyState/EmptyState.tsx

```diff
@@ -0,0 +1,124 @@
+import styled from 'styled-components';
```

Comment:
i may abstract more here and/or abstract away the logic for the automated slider into a hook as well, but I want to finish the device trust empty state to see whats needed first